Formula/pastebinit.rb

```diff
@@ -25,6 +25,9 @@ def install
   end
 
   test do
-    system "date | pastebinit"
+    url = shell_output("echo \"Hello world at $(date)\" | pastebinit").chomp
+    assert_match %r{^http://pastebin\.com/}, url
+    # We can't actually fetch the URL to check the paste's success because pastebin
+    # blocks our fetches with curl, probably based on the user agent
```

Comment:
`curl` can set its own user agent with `-A/--user-agent` option, so that can probably be defeated without too much trouble.